src/qiboconnection/models/job_response.py

```diff
@@ -0,0 +1,45 @@
+# Copyright 2023 Qilimanjaro Quantum Tech
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+""" JobResponse """
+from dataclasses import dataclass
+
+from qiboconnection.util import from_kwargs
+
+from ..typings.requests import JobRequest
+
+
+@dataclass
+class JobResponse(JobRequest):
+    """Full Job Response. Includes job results which may
+    be weight a few GB.
+
+    Attributes:
+        user_id (int): User identifier
+        device_id (int): Device identifier
+        description (str): Description of the job
+        job_id (int): Job identifier
+        queue_position (int): Job queue position
+        status (str): Status of the job
+        result (str): Job result
+    """
+
+    job_id: int
+    queue_position: int
+    result: str
+    status: str
+
+    @classmethod
+    def from_kwargs(cls, **kwargs):
+        "Returns an instance of JobResponse including non-typed attributes"
+        return from_kwargs(cls, **kwargs)
```

Comment:
Now that I think of this, we could have a mixin-based approach.

Desing a class --lets call it `BuildableFromKwargs` for this exaple's sake-- that has simply this specific functionality. Then let every class that wants to implement it inherit from whatever it needs to plus BuildableFromKwargs.

With this, we wouldn't have to implement the  `def from_kwargs()` at so many places.

Reply:
this mat be a good idea? However, am I implementing `def from_kwargs()`? To my understanding, I only implemented it once in `util.py`